### PR TITLE
Homescreen UI Addition

### LINKED ISSUE #11: Develop HomePage Widget
I hope we should also design a homepage for the application. Can you please review this issue and assign it to me?

### PR DESCRIPTION
**Pull request by Praneeth-rdy**
Fixes #11 

### Changes
- Added navigator push in login screen after successful login
- Added google font dependency
- Added images in assets
- Added homescreen UI in homescreen.dart

Homescreen UI developed is:

hm-UI

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -33,6 +33,13 @@ dependencies:
   hive_flutter: ^1.0.0
   lint: ^1.0.0
   path_provider: ^2.0.1
+  hive: ^2.0.0
```
Comment (pratyushmp) [inline pubspec.yaml:36]: 'hive' and 'cupertino icon' packages are already declared. Remove the duplicate packages

**Review by pratyushmp — CHANGES_REQUESTED** [review]
There are couple of spelling mistakes in the introscreen as well as the firebase verification dialogue box which needs to be fixed!

**Review by adityathakurxd — COMMENTED** [review]
Looks good to me. Please look into the changes requested by @pratyushmp 
Remove the duplicates.

**Comment by Praneeth-rdy** [comment]
Made the changes mentioned above.

**Review by pratyushmp — APPROVED** [review]

**Comment by infiniteoverflow** [comment]
Looks wonderful. Good job !!

**Pull request merged**